dvc/command/experiments.py

```diff
@@ -122,6 +134,30 @@ def _collect_rows(
         yield row, style
 
 
+def _sort_exp(experiments, sort_by, typ, reverse):
+    if "baseline" in experiments:
+        ret = OrderedDict({"baseline": experiments.pop("baseline")})
+    else:
+        ret = OrderedDict()
+
+    def _sort(item):
+        _, exp = item
+        for fname, item in exp.get(typ, {}).items():
+            if isinstance(item, dict):
+                item = flatten(item)
+            else:
+                item = {fname: item}
+            if sort_by in item:
+                val = item[sort_by]
+                break
+        else:
+            val = None
+        return (val is None, val)
```

Comment:
for-else is a bit confusing. `return` based will be more readable.
```suggestion
            if sort_by in item:
                val = item[sort_by]
                return (val is None, val)
        return (True, None)
```